likecoin/admin/metabox.php

```diff
@@ -352,13 +387,24 @@ function likecoin_add_meta_box( $post, $button_params, $publish_params ) {
 	<div class="wrapper">			
 		<?php likecoin_add_button_meta_box( $button_params ); ?>
 		<hr>
-		<?php likecoin_add_publish_meta_box( $publish_params ); ?>
+		<?php likecoin_add_publish_meta_box( $publish_params, $post ); ?>
 	</div>
 	<?php
-		$post_id    = $post->ID;
-		$post_title = $post->post_title;
-		$post_tags  = likecoin_get_post_tags_for_matters( $post );
-		$post_url   = get_permalink( $post );
+		$post_id           = $post->ID;
+		$post_title        = $post->post_title;
+		$post_tags         = likecoin_get_post_tags_for_matters( $post );
+		$post_url          = get_permalink( $post );
+		$matters_ipfs_hash = $publish_params['ipfs_hash'];
+		$arweave_info      = get_post_meta( $post_id, LC_ARWEAVE_INFO, true );
+		$arweave_id        = '';
+		$arweave_ipfs_hash = '';
+	if ( ! is_array( $arweave_info ) ) {
+		$arweave_id        = '';
+		$arweave_ipfs_hash = '';
+	} else {
+		$arweave_id        = $arweave_info['arweave_id'];
+		$arweave_ipfs_hash = $arweave_info['ipfs_hash'];
+	}
```

Comment:
Why repeat
```
$arweave_id        = '';
$arweave_ipfs_hash = '';
```
twice?

Reply:
True. revered the condition to avoid redundancy.